cli/Sources/TuistKit/Commands/Cache/CacheStartCommand.swift

```diff
@@ -9,28 +9,21 @@ struct CacheStartCommand: AsyncParsableCommand {
         shouldDisplay: false
     )
 
+    @Argument(
+        help: "The full handle of the project (account-handle/project-handle)."
+    )
+    var fullHandle: String
```

Comment:
I'm changing the interface of this one, so we the launch configuration for the daemon doesn't depend on the project's path